web/src/main/java/no/nav/foreldrepenger/web/app/tjenester/behandling/uttak/dto/UttakResultatPerioderDto.java

```diff
@@ -12,7 +17,26 @@ public record UttakResultatPerioderDto(@NotNull List<UttakResultatPeriodeDto> pe
                                        boolean annenForelderRettEØS,
                                        boolean oppgittAnnenForelderRettEØS,
                                        @NotNull FilterDto årsakFilter,
-                                       @NotNull LocalDate endringsdato) {
+                                       @NotNull LocalDate endringsdato,
+                                       @NotNull List<PeriodeResultatÅrsakDto> muligeÅrsaker) {
 
     public record FilterDto(@NotNull LocalDate kreverSammenhengendeUttakTom, @NotNull boolean utenMinsterett, @NotNull boolean søkerErMor) {}
+
+    public record PeriodeResultatÅrsakDto(String kode,
```

Comment:
Hva er @NotNull her - for kodegenerering. 
OBS på om UDEFINERT skal med eller ikke. Tror ikke den er med i dag.